lib/fs.js

```diff
@@ -2379,8 +2379,13 @@ WriteStream.prototype.open = function() {
 
 
 WriteStream.prototype._write = function(data, encoding, cb) {
-  if (!(data instanceof Buffer))
-    return this.emit('error', new Error('Invalid data'));
+  if (!(data instanceof Buffer)) {
+    const err = new errors.TypeError('ERR_INVALID_ARG_TYPE',
+                                     'data',
+                                     'Buffer',
+                                     data);
+    return this.destroy(err);
```

Comment:
The CI seems failed due to this line. It should be:
```js
return this.emit('error', err);
```
I've tested it on my environment and passed all the test 😉

Reply:
That’s what it was before but @mcollina said to use destroy instead.
I will change it back.
Thanks for the heads up 😎

Reply:
@starkwang How does that look?